lib/gyp/generator/ninja/index.js

```diff
@@ -350,8 +353,8 @@ Ninja.prototype.actionCmd = function actionCmd(base, toBase, cmds) {
   if (this.flavor !== 'win32')
     return res;
 
-  // TODO(indutny): escape quotes in res
-  return `cmd.exe /s /c "${res}"`;
+  const nw = gyp.platform.win.getNinjaWrapper(this.targetArch);
+  return `${nw} cmd.exe /s /c "${res}"`;
```

Comment:
Yep. also `res = 'xxx "yyy" zzz'`.
as long as `res` is [well formed](https://nodejs.org/api/child_process.html#child_process_spawning_bat_and_cmd_files_on_windows)